plugin/storage/badger/options.go

```diff
@@ -44,6 +45,9 @@ type NamespaceConfig struct {
 	ReadOnly              bool          `mapstructure:"read_only"`
 }
 
+// TODO deprecated flag to be removed
+const truncateWarning = "(deprecated since v1.24.0, will be removed in release v1.26.0)"
```

Comment:
nit: it may be worth including the time dimension too (3 months notice) as in the example from [CONTRIBUTING.md](https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md#deprecating-cli-flags), which would be more pertinent if we decide to have more frequent releases; or did we decide to only use release versions to determine the deprecation notice period?